app/src/main/java/com/hedvig/app/feature/embark/EmbarkViewModel.kt

```diff
@@ -41,17 +42,25 @@ abstract class EmbarkViewModel(
     protected lateinit var storyData: EmbarkStoryQuery.Data
 
     private val store = HashMap<String, String>()
+    private lateinit var computedValues: Map<String, String>
```

Comment:
```suggestion
    private val computedValues: HashMap<String, String> = hashMap()
```

We can populate it when we set the initial state, later on 🙂